packages/documentation/src/pages/getting-started/common-tasks/new-application.mdx

```diff
@@ -30,6 +30,20 @@ Here are the steps for manually setting up a new application. We're going to wal
   // The Webpack bundle name for your app. Must be unique.
   "entryName": "my-new-application",
   // The root url where your application lives
+  // This one is only used by webpack-dev-server for local development
+  "rootUrl": "/my-new-application",
+}
+```
+
+3. Add an entry to `src/applications/registry.json`. This file contains information that the content build needs to create the landing page for your application:
```

Comment:
This is another sneaky diff. I removed parts of the `manifest.json` and created a new step for adding an entry in `registry.json`.